### PR TITLE
Sixth Korean l10n work for release 1.18

### PR DESCRIPTION
**Pull request by gochist**
- Update outdated files in dev-1.18-ko.6 partly (#21714)
- Translate StorageClass to Korean word (#21805)
- Translate StatefulSet to Korean word in pods.md (#21794)
- Translate tasks/run-application/delete-stateful-set/ in Korean (#21686)
- Translate tasks/administer-cluster/change-default-storage-class in Korean (#21801)
- update outdated docs (#21940)
- Modify spacing term StatefulSet in Korean (#21871)
- Translate /tasks/administer-cluster/coredns in Korean (#21876)
- Fix typo in k8s.io/ko/docs/contribute/new-content/new-content/ (#21975)
- Translation error correction in /concepts/containers/runtime-class.md (#21868)
- Translate tasks/tls/certificate-rotation/ in Korean (#21838)
- Translate /tasks/configure-pod-container/static-pod in Korean (#21798)
- Update to Outdated files in the dev-1.18-ko.6 branch - (1/4) (#21911)
- Fix English bugs in Korean documentation (#21994)
- Update outdated dev-1.18-ko.6 partly (#22067)

Co-authored-by: Jerry Park <jaehwa@gmail.com>
Co-authored-by: PyungHo Yoon <learder@gmail.com>
Co-authored-by: Yuk, Yongsu <ysyukr@gmail.com>
Co-authored-by: Jerry Park <jaehwa@gmail.com>
Co-authored-by: Seokho Son shsongist@gmail.com
Co-authored-by: Jordy Ruiter <jordy.ruiter@gmail.com>
Co-authored-by: bluefriday <bluefriday86@gmail.com>
Co-authored-by: Dajin Gwon <d.gweon@samsung.com>
Co-authored-by: Hyungseok Lee <hs0426.lee@samsung.com>
Co-authored-by: Jihoon Seo <jihoon.seo@etri.re.kr>
Co-authored-by: coolguyhong <podolsmith@naver.com>
Co-authored-by: jmyung <jesang.myung@gmail.com>
Co-authored-by: June Yi <june.yi@samsung.com>

### CODE REVIEW

File: content/ko/docs/contribute/localization_ko.md [hunk 1]
```diff
@@ -386,29 +412,33 @@ redirect | 리다이렉트(redirect) |
...
-Secret | 시크릿 |
+Secret | 시크릿(Secret) | API 오브젝트인 경우
 segment | 세그먼트 |
 Selector | 셀렉터 |
 Self-healing | 자가 치유 |
-Service | 서비스 |
-Service Account | 서비스 어카운트 |
+SelfSubjectAccessReview | 셀프서브젝트액세스리뷰(SelfSubjectAccessReview) | API 오브젝트인 경우
+SelfSubjectRulesReview | SelfSubjectRulesReview | API 오브젝트이지만 용어를 구성하는 단어 중 복수형 Rules를 '룰스'로 외래어 표기하는 경우 한국어 독자에게 다소 생경할 수 있어 예외적으로 영문 용어를 사용함
+Service | 서비스 | API 오브젝트인 경우에도 표현의 간결함을 위해 한영병기를 하지 않음
```
Comment (sftim) [inline content/ko/docs/contribute/localization_ko.md:440]: I'm surprised not to see
```suggestion
Service | 서비스(Service) | API 오브젝트인 경우에도 표현의 간결함을 위해 한영병기를 하지 않음
```

_but_ it's fine to merge as-is, of course.

Comment (pjhwa) [reply content/ko/docs/contribute/localization_ko.md:440]: @sftim For "Pod" and "Service", it was decided not to use Korean-English notation. So, we wrote in the remarks like the below:
"Even in the case of API objects, Korean-English notation are not used for simplicity of expression"

File: content/ko/docs/tasks/run-application/delete-stateful-set.md [hunk 2]
```diff
@@ -0,0 +1,88 @@
...
+
+`kubectl delete` 에 `--cascade=false` 를 사용함으로써, 스테이트풀셋 객체가 삭제 된 후에도 스테이트풀셋에 의해 관리된 파드는 남게 된다. 만약 파드가 `app=myapp` 레이블을 갖고 있다면, 다음과 같이 파드를 삭제할 수 있다.
+
+```shell
+kubectl delete pods -l app=myapp
+```
+
+### 퍼시스턴트볼륨(PersistentVolume)
+
+스테이트풀셋의 파드들을 삭제하는 것이 연결된 볼륨을 삭제하는 것은 아니다. 이것은 볼륨을 삭제하기 전에 볼륨에서 데이터를 복사할 수 있는 기회를 준다. 파드들이 [terminating 상태](/ko/docs/concepts/workloads/pods/pod/#termination-of-pods)가 된 후 PVC를 삭제하는 것은 스토리지클래스(StorageClass) 와 반환 정책에 따라 백업 퍼시스턴트볼륨이 삭제될 수도 있다. 클레임 삭제 후 볼륨에 접근할 수 있다고 가정하면 안된다.
```
Comment (pjhwa) [inline content/ko/docs/tasks/run-application/delete-stateful-set.md:55]:
```suggestion
스테이트풀셋의 파드들을 삭제하는 것이 연결된 볼륨을 삭제하는 것은 아니다. 이것은 볼륨을 삭제하기 전에 볼륨에서 데이터를 복사할 수 있는 기회를 준다. 파드들이 [terminating 상태](/ko/docs/concepts/workloads/pods/pod/#파드의-종료)가 된 후 PVC를 삭제하는 것은 스토리지클래스(StorageClass) 와 반환 정책에 따라 백업 퍼시스턴트볼륨이 삭제될 수도 있다. 클레임 삭제 후 볼륨에 접근할 수 있다고 가정하면 안된다.
```

- URL에 오류가 있습니다.

File: content/ko/docs/tutorials/stateless-application/guestbook-logs-metrics-with-elk.md [hunk 3]
```diff
@@ -406,4 +405,3 @@ kubectl scale --replicas=3 deployment/frontend
 * [로깅 아키텍처](/docs/concepts/cluster-administration/logging/)를 더 읽어본다.
```
Comment (pjhwa) [inline content/ko/docs/tutorials/stateless-application/guestbook-logs-metrics-with-elk.md:405]:
```suggestion
* [로깅 아키텍처](/ko/docs/concepts/cluster-administration/logging/)를 더 읽어본다.
```

- 번역된 문서가 있습니다.

File: content/ko/docs/tutorials/stateless-application/guestbook.md [hunk 4]
```diff
@@ -369,4 +369,3 @@ Google Compute Engine 또는 Google Kubernetes Engine과 같은 일부 클라우
 * [애플리케이션 접속](/ko/docs/concepts/services-networking/connect-applications-service/)에 대해 더 알아보기
 * [자원 관리](/docs/concepts/cluster-administration/manage-deployment/#using-labels-effectively)에 대해 더 알아보기
```
Comment (pjhwa) [inline content/ko/docs/tutorials/stateless-application/guestbook.md:370]:
```suggestion
* [자원 관리](/ko/docs/concepts/cluster-administration/manage-deployment/#효과적인-레이블-사용)에 대해 더 알아보기
```

- 번역된 문서가 있습니다.

**Comment by gochist** [comment]
@pjhwa 리뷰 감사합니다. 다만, 이 PR의 목적은 ko.6 마일스톤 동안 공동의 작업을 master 브랜치에 단일 커밋으로 반영하고 ko.7을 시작할 수 있도록 하는 것이기에, 마일스톤 진행 중 공동의 작업이 누락 없이 반영되었는지 여부에 리뷰 초점을 맞추고, 지적해주신 내용은 ko.7 마일스톤에서 반영될 수 있도록 새로운 이슈나 PR을 여는 것이 좋을 것 같습니다.

**Pull request merged**